components/full-width-header/full-width-header.js

```diff
@@ -0,0 +1,23 @@
+import Icon from '../icon/icon';
+import getDateFormat from '../_helpers/getDateFormat';
+
+const FullWidthHeader = ({header}) => (
```

Comment:
I would separate all content and add them as separate attributes on the component. So `title` , `color`, `image` etc